subprojects/common/core/src/main/java/hu/bme/mit/theta/core/type/fptype/FpType.java

```diff
@@ -155,4 +141,9 @@ public GtExpr<FpType> Gt(Expr<FpType> leftOp, Expr<FpType> rightOp) {
     public GeqExpr<FpType> Geq(Expr<FpType> leftOp, Expr<FpType> rightOp) {
         return FpExprs.Geq(leftOp, rightOp);
     }
+
+    @Override
+    public DomainSize getDomainSize() {
+        return DomainSize.INFINITY;
```

Comment:
Technically, this is not true. We are storing the value on finite amount of bits.

Reply:
Should be fixed with a315e11, please re-check